### PR TITLE
Simplify build for MSVC

### PR DESCRIPTION
**Pull request by EhWhoAmI**
MSVC creates different directories for Debug and release builds, and it does not copy all the necessary DLLs to the same directory. 

This reduces the need to copy files manually after each build.

### CODE REVIEW

**Comment by EhWhoAmI** [comment]
I'm gonna skip adding extra command line arguments because I have no idea how to do that.

**Review by jonesmz — APPROVED** [review]
As I said on discord, aside from considering the need to make these changes to be really odd, i don't see anything immediately obviously wrong, so I won't object to this being merged.

@z-adams or @Capital-Asterisk I'll leave it to one of you to merge this.

**Review by z-adams — CHANGES_REQUESTED** [review]
I'd like to tweak this stuff so don't merge it yet please, I just haven't had time on my desktop to test it.

**Comment by Capital-Asterisk** [comment]
I have never built on windows before, but I have a branch that sets up the CI to uploads binaries when finished, this might be interesting:

https://github.com/Capital-Asterisk/osp-magnum/tree/release-artifact

I had to manually merge together `build/bin/Release/*` and `build/Release/lib`, and add `build/bin/OSPData` to get it working (on wine). I'm assuming this change will make this unnecessary, which is great for distribution.

**Comment by EhWhoAmI** [comment]
Yes, it should reduce the need to manually merge the folders together, which is extra annoying when you want a new build.

**Review by z-adams — APPROVED** [review]

**Pull request merged**
